src/main/java/org/zalando/failsafeactuator/endpoint/FailsafeEndpoint.java

```diff
@@ -23,15 +23,47 @@
 
   private static final String ENDPOINT_ID = "failsafe";
   private final CircuitBreakerRegistry circuitBreakerRegistry;
+  private final ApplicationContext context;
 
-  public FailsafeEndpoint(final CircuitBreakerRegistry circuitBreakerRegistry) {
+  public FailsafeEndpoint(final CircuitBreakerRegistry circuitBreakerRegistry, final ApplicationContext context) {
```

Comment:
Can't you just inject the `CircuitBreaker` instances via `Map<String, CircuitBreaker`?

Reply:
I'd prefer to stick with this implementation since the benefit is probably rather low I think?